opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -37,6 +38,12 @@
 
 namespace Opm {
 
+    /// Returns a copy of s where every letter has been upper cased.
+    static std::string toupper(const std::string& s) {
+        auto ret = std::string(s);
+        std::transform(ret.begin(), ret.end(), ret.begin(), ::toupper);
```

Comment:
> and friends from C can be implemented as macros 

I am not claming this is wrong - but I have not found any reference to a macro based implementation in cpp ref?